base/base_tests/bits_test.cpp

```diff
@@ -143,10 +144,10 @@ UNIT_TEST(NumUsedBits)
 
 UNIT_TEST(PopCount64)
 {
-  TEST_EQUAL(0, bits::PopCount(static_cast<uint64_t>(0x0)), ());
-  TEST_EQUAL(1, bits::PopCount(static_cast<uint64_t>(0x1)), ());
-  TEST_EQUAL(32, bits::PopCount(static_cast<uint64_t>(0xAAAAAAAA55555555)), ());
-  TEST_EQUAL(64, bits::PopCount(static_cast<uint64_t>(0xFFFFFFFFFFFFFFFF)), ());
+  TEST_EQUAL(0, std::popcount(static_cast<uint64_t>(0x0)), ());
```

Comment:
There is no need to test standard library functions. We trust them )

Reply:
Trust, but verify :)